WindowsServerDocs/get-started/extended-security-updates.md

```diff
@@ -41,6 +41,9 @@ For more information, see the [Extended Security Updates frequently asked questi
 
 If you run Windows Server 2008 or 2008 R2 VMs in Azure, they're automatically enabled for Extended Security Updates. You don't need to configure anything, and there's no additional charge for using Extended Security Updates with Azure VMs. Extended Security Updates are automatically delivered to Azure VMs if they're configured to receive updates.
 
+> [!NOTE]
+> Microsoft.ClassicCompute type VMs require additional configuration for ESU deployment, as they do not have access to the Azure Resource Manager, in particular the [Azure Instance Metadata Service](https://docs.microsoft.com/azure/virtual-machines/windows/instance-metadata-service), which is used for evaluating ESU eligibility. Please contact [Microsoft support](http://support.microsoft.com/contactus?PID=17336) for more help.
```

Comment:
@andreiztm  Please change the following: 
- ESU to Extended Security Updates. For some reason, even though we show the acronym, it's always spelled out completely in the text.
- "Do not have" to "don't have"

I'm unclear whether your are a Microsoft employee and subject matter expert on this, or if we need to have someone review this text before publishing. Can you clarify your role? Thanks!